gsy_framework/utils.py

```diff
@@ -544,3 +544,21 @@ def sort_list_of_dicts_by_attribute(input_list: List[Dict], attribute: str, reve
 def convert_datetime_to_ui_str_format(data_time):
     """Convert a datetime object into the format required by the UI."""
     return instance(data_time).format(DATE_TIME_UI_FORMAT)
+
+
+def is_time_slot_in_simulation_duration(
```

Comment:
Please also create a follow up PR for the gsy-e where you use this function. Thanks a lot!